confluent_kafka/src/confluent_kafka.c

```diff
@@ -1422,7 +1422,7 @@ static PyObject *libversion (PyObject *self, PyObject *args) {
 }
 
 static PyObject *version (PyObject *self, PyObject *args) {
-	return Py_BuildValue("si", "0.9.2", 0x00090100);
+	return Py_BuildValue("si", "0.9.5", 0x00090100);
```

Comment:
This should probably still be at 0.9.4, and the hex version needs to be updated at as well 0x00090400